upstairs/src/client.rs

```diff
@@ -938,8 +922,11 @@ impl DownstairsClient {
             // We never send jobs if we're in certain inactive states
             DsState::Faulted
             | DsState::Replaced
-            | DsState::Replacing
-            | DsState::LiveRepairReady => EnqueueResult::Skip,
+            | DsState::LiveRepairReady
+            | DsState::Stopping(..) => EnqueueResult::Skip,
+            // XXX there are some `Stopping` cases which should never happen,
+            // should we panic on them (like we do for negotiation states
+            // below)?
```

Comment:
I'd say yes but I think that should be addressed in the next PR in the series (?)

Reply:
I tried both ways, but because it's just affecting `DsState::Stopping(..)`, it seems reasonable to do here.